.rubocop.yml

```diff
@@ -11,4 +11,12 @@ Style/Documentation:
 
 Metrics/BlockLength:
   Exclude:
-    - 'spec/**/*'
\ No newline at end of file
+    - 'spec/**/*'
+
+Style/StringLiterals:
```

Comment:
We actually use Standard so this config shouldn't really be here, even though Standard uses Rubocop under the hood.

https://github.com/standardrb/standard